azure-iot-provisioning-devicesdk/azure/iot/provisioning/devicesdk/transport/symmetric_key_transport.py

```diff
@@ -0,0 +1,24 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class SymmetricKeyTransport(object):
```

Comment:
As per our discussion I'd like to see this class removed for the time being, and a solution for handling different methods of security client authentication added only when necessary, and the problem has been fully scoped. (the next PR with X509)

Reply:
i am not using this anywhere. I have kept this file around 

---
In reply to: [269801835](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r269801835) [](ancestors = 269801835)